imagetest/test_suites/network/setup.go

```diff
@@ -0,0 +1,16 @@
+package network
+
+import "github.com/GoogleCloudPlatform/guest-test-infra/imagetest"
+
+// Name is the name of the test package. It must match the directory name.
+var Name = "network"
+
+// TestSetup sets up the test workflow.
+func TestSetup(t *imagetest.TestWorkflow) error {
+	vm, err := t.CreateTestVM("vm")
+	if err != nil {
+		return err
+	}
+	vm.RunTests("TestDefaultMTU")
```

Comment:
nitpick: you don't need RunTests unless you're filtering. the default is that all tests will run.